src/App.css

```diff
@@ -0,0 +1,38 @@
+.App {
```

Comment:
also css file name should not be capitalized. Also I dont see this file being used in App.js folder.